components/automate-gateway/gateway/datacollector_test.go

```diff
@@ -169,7 +169,7 @@ func TestDataCollectorParseBytesToChefRunFailure(t *testing.T) {
 	assert.Equal(t, "", res1.Delta)
 	assert.Equal(t, "insights-test", res1.CookbookName)
 	assert.Equal(t, "0.1.1", res1.CookbookVersion)
-	assert.Equal(t, false, res1.IgnoreFailure.GetBoolValue())
+	assert.Equal(t, true, res1.IgnoreFailure.GetBoolValue())
```

Comment:
This is the only real change to this test. The other tests are the go simplification.